doc/guide/dynamics/dynamics-bloch-redfield.rst

```diff
@@ -226,15 +226,17 @@ Note that it is also possible to add Lindblad dissipation superoperators in the
 
     H = - delta/2.0 * sigmax() - eps0/2.0 * sigmaz()
 
-    def ohmic_spectrum(w):
+    def ohmic_spectrum(_, w):
```

Comment:
Having to specify `_` in order to receive and ignore `t` seems a bit odd, but that is unrelated to this PR.